python/cudf/cudf/tests/test_parquet.py

```diff
@@ -3902,24 +3902,79 @@ def test_parquet_chunked_reader_string_decoders(
 
 
 @pytest.mark.parametrize(
-    "nrows,skip_rows",
+    "nrows, skip_rows",
```

Comment:
Can we add a test is `skip_rows> len(df)`? Do we just return an empty dataframe? I'd be fine with adding this to the C++ test suite if you prefer.

Reply:
Yeah of course. Yes we return an empty df. Adding one

Reply:
Added in ebc5442